inc/functions/files.php

```diff
@@ -622,7 +622,7 @@ function rocket_clean_files( $urls, $filesystem = null ) {
 
 	/** This filter is documented in inc/front/htaccess.php */
 	$url_no_dots = (bool) apply_filters( 'rocket_url_no_dots', false );
-	$cache_path  = rocket_get_constant( 'WP_ROCKET_CACHE_PATH' );
+	$cache_path  = _rocket_get_wp_rocket_cache_path();
```

Comment:
Why you defined the functions with a leading _ ? Is there a benefit / purpose of using it like this?

Reply:
Just saw the conversation is slack.

Reply:
To capture the conversation here:

These are private functions which are for internal use only. We don’t want 3rd parties using them.

Benefits to using this naming convention and marking them private:

- Recognizable from other programming languages as “private” for internal use. It’s a common function naming convention in other languages. It’s effective because you stop and wonder why before using it.

- We won’t need to deprecate them later when we build the Filesystem module,